doc/level1.rst

```diff
@@ -980,4 +980,12 @@ Example Setup Message User Interaction
 
 .. todo::
 
-   TODO
+   Is fine just with this?
+
+.. this code decrypts the example Autocrypt Setup Message
+
+::
+
+    This is your Setup Code to decrypt the Autocrypt Setup Message:
+
+    1742-0185-6197-1303-7016-8412-3581-4441-0597
```

Comment:
I think we're recommending to present the setup code to the user in the grid format, for easier transcription and copying.